tox.ini

```diff
@@ -6,32 +6,49 @@
 ;     # Only runs the linter checks
 ;   $ tox -e py38
 ;     # Runs all tests against python3.8 only
-;   $ tox -e py
-;     # Runs tests against the python interpreter where tox itself was installed
+;   $ export TOXENV=py39
+;   $ tox
+;     # If present use $TOXENV environment variable
 envlist = lint, py3{6,7,8,9}
```

Comment:
We now have at least two enumerations of supported python version, none of which are obvious. Can we avoid that somehow?

Reply:
Even just a single file that enumerates them that everyone refers to?

Reply:
the source of truth is now tox.ini's `envlist`. Tox allows to list the environments from its envlist with `tox -l` command. I now iterate over them and run pip-compile for each one.